sdk/containerregistry/container-registry/README.md

```diff
@@ -137,10 +137,10 @@ async function main() {
   const image = client.getArtifact("library/hello-world", "latest");
 
   // List the set of tags on the hello_world image tagged as "latest"
-  const tags = image.listTagPropertis();
+  const tags = image.listTagProperties();
 
   // Iterate through the image's tags, listing the tagged alias for the image
-  console.log(`${image.fullyQualifiedName}  has the following aliases:`);
+  console.log(`${image.fullyQualifiedReference}  has the following aliases:`);
   for await (const tag of tags) {
```

Comment:
nit: I feel like it would be clearer that `listTagProperties` returns an async iterator. I only brought this up because the current samples had me asking myself what `listTagProperties` returns

Reply:
Previously for other SDK lib we got feedback from ArchBoard to take the `list*()` function call out of the `for-await-of` and introduce a local variable for it.

Reply:
Although `tagIterator` would be a better local name. I will rename.